src/locale/fr-FR.h

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 - Mathieu D(@Knackie)
+// Locale/language file for French / Quebec.
+// This file will override the default values located in `defaults.h`.
+#ifndef LOCALE_FR_FR_H_
+#define LOCALE_FR_FR_H_
+
+#define D_STR_UNKNOWN "INCONNU"
+#define D_STR_PROTOCOL "Protocole"
+#define D_STR_ON "On"
+#define D_STR_OFF "Off"
+#define D_STR_MODE "Mode"
+#define D_STR_TOGGLE "Bascule"
+#define D_STR_SLEEP "Pause"
+#define D_STR_LIGHT "Lumière"
+#define D_STR_POWERFUL "Puissance"
+#define D_STR_QUIET "Silence"
+#define D_STR_ECONO "Economie"
+#define D_STR_BEEP "Bip"
+#define D_STR_MOLD "Moule"
+#define D_STR_CLEAN "Nettoyer"
+#define D_STR_PURIFY "Purifier"
+#define D_STR_TIMER "Timer"
+#define D_STR_ONTIMER D_STR_ON " " D_STR_TIMER
+#define D_STR_OFFTIMER D_STR_OFF " " D_STR_TIMER
+#define D_STR_CLOCK "Heure"
+#define D_STR_COMMAND "Commandement"
+#define D_STR_HEALTH "Santé"
+#define D_STR_TEMP "Temporaire"
+#define D_STR_HUMID "Humidité"
+#define D_STR_SAVE "Sauvegarder"
+#define D_STR_EYE "Oeil"
+#define D_STR_FOLLOW "Suivre"
+#define D_STR_FRESH "Frais"
+#define D_STR_HOLD "Maintenir"
+#define D_STR_BUTTON "Bouton"
+#define D_STR_NIGHT "Nuit"
+#define D_STR_SILENT "Silence"
+#define D_STR_UP "En haut"
+#define D_STR_TEMPUP D_STR_TEMP " " D_STR_UP
+#define D_STR_DOWN "En bas"
+#define D_STR_TEMPDOWN D_STR_TEMP " " D_STR_DOWN
+#define D_STR_CHANGE "Changement"
+#define D_STR_MOVE "Move"
+#define D_STR_SET "Mettre"
+#define D_STR_CANCEL "Annuler"
+#define D_STR_COMFORT "Cpnfort"
```

Comment:
Shouldn't this be "Confort"? e.g. a typo?

Reply:
My fault, it's a typo yes !